corehq/apps/locations/models.py

```diff
@@ -116,6 +116,18 @@ class LocationQueriesMixin(object):
     def location_ids(self):
         return self.values_list('location_id', flat=True)
 
+    def couch_locations(self, wrapped=True):
+        """
+        Returns the couch locations corresponding to this queryset.
+        If you're calling this, take a long hard look at yourself and ask why.
+        Do you really need couch locations here?
+        """
+        ids = self.location_ids()
+        locations = iter_docs(Location.get_db(), ids)
+        if wrapped:
+            return map(Location.wrap, locations)
+        return locations
```

Comment:
it's slightly confusing that one fork of this function returns a `list` while another returns an `iterator`